pkg/dom0-ztools/Dockerfile

```diff
@@ -42,3 +43,18 @@ FROM scratch
 COPY --from=zfs /out/ /
 # hadolint ignore=DL3020
 ADD rootfs/ /
+### Create system-wide groups and users ###
+# add initial root user
+RUN touch /etc/group && touch /etc/passwd
+RUN addgroup -g 0 root
+RUN adduser -D -H -h /root -s /bin/sh -g "root" -G root -u 0 root
+# add nobody user and group
```

Comment:
What is the nobody user? How is it expected to be used?

Reply:
It is user that basically owns nothing, the common use case is to acquire the resources a process needs (like sockets, fds, etc) and then switch to nobody user at run-time to have basically no access so in case process gets compromised it can't lead to exposing other resources. 

You can do some tricks with it too, for example ssh makes binary setgid nobody, so when it is running even the current user (with equal uid to ssh process) can't ptrace the ssh process and dump keys and secrets ...

Reply:
Could we document it a little? In some README.md. So, we later know what the idea was and how it should be used.

Reply:
it is pretty well know, even has a wikipedia entry https://en.wikipedia.org/wiki/Nobody_(username)